### DISCUSSION TITLE
Dns At school

### DISCUSSION THREAD

**Discussion by pazza2234** [3kh0/ext-remover#760] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I have full rootfs acsess with fakemurk and murkmod but for the life of me I can not change the school dns without removing it from policy's and manualy adding it. witch I can not do because if I manualy add it it wont connect to anything period. so I nead a way without changing the deviceopennetworkconfig (or open networkconfig) to change the dns. I CAN DO BASIClY ANYTHING ELSE. 

thank you.

**Comment by S-PScripts** [comment]
why do you want to do this xd

anyways stale discussion im closing it
